File: BuddySystem.Models/Trip/TripListItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BuddySystem.Models
{
    public class TripListItem
    {
        public int TripId { get; set; }
        public string StartLocation { get; set; }
        public string EndLocation { get; set; }
        public string Description { get; set; }
        public int PrimaryBuddyId { get; set; }
        public string PrimaryBuddyName { get; set; }
        public int VolunteerId { get; set; }
        public string VolunteerName { get; set; }

    }
}

File: angular-buddy-system/src/app/models/TripListItem.ts
/*This model's sole purpose is to display some trip details for a list */
export interface TripListItem {
    TripId?: number;
    StartLocation: string;
    EndLocation: string;
    Description: string;
    BuddyId: number;
    BuddyName: string;
    VolunteerId?: number;
    VolunteerName?: string;
}